admin/warehouses/V1.11.0__create_dpe_automation_warehouse.sql

```diff
@@ -0,0 +1,7 @@
+use role sysadmin;
+create warehouse if not exists dpe_automation
```

Comment:
Should we go through this repo and re-point any of the scripts or schema changes to use this warehouse at this point in time?